### PR TITLE
beta of Jakarta Data

### PR DESCRIPTION
**Pull request by njr-11**
The Jakarta Data specification released a first beta. We should do the same with our data–1.0 feature so that users can experiment with the specification.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.ws.persistence/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -36,6 +36,9 @@ dataSoure.desc=Data source that connects to the persistent store.
 createTables=Create database tables
 createTables.desc=When set to true, database tables are created.
 
+dropTables=Drop database tables
```
Comment (dmuelle) [inline dev/com.ibm.ws.persistence/resources/OSGI-INF/l10n/metatype.properties:39]:
```suggestion
dropTables=Drop database tables.
```

File: dev/com.ibm.ws.persistence/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -36,6 +36,9 @@ dataSoure.desc=Data source that connects to the persistent store.
...
+dropTables.desc=When set to true, database tables are removed. When both createTables and dropTables are set to true, tables are removed first and then created. This configuration property is intended for use with the Beta of Jakarta Data, to provide a way to remove old tables from the database when changes are made to entity classes. At a minimum, you must restart the application for changes to this property to go into effect.
```
Comment (dmuelle) [inline dev/com.ibm.ws.persistence/resources/OSGI-INF/l10n/metatype.properties]:
```suggestion
dropTables.desc=When this property is set to true, database tables are removed. When both the createTables and dropTables properties are set to true, existing tables are removed first, and then new tables are created. This configuration property is intended for use with the beta of Jakarta Data, to provide a way to remove old tables from the database when changes are made to entity classes. At a minimum, you must restart the application for changes to this property to go into effect.
```

**Review by dmuelle — COMMENTED** [review]
ID review

File: dev/com.ibm.ws.persistence/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (njr-11) [reply dev/com.ibm.ws.persistence/resources/OSGI-INF/l10n/metatype.properties:39]: @dmuelle The recommended change is not correct. This is the name of a configuration property, and these names never and with a period.  Refer to other examples within this file, such as `createTables=Create database tables` and `dataSource=Data Source` which are right above this.

Comment (dmuelle) [reply dev/com.ibm.ws.persistence/resources/OSGI-INF/l10n/metatype.properties:39]: ok- sorry- I misread this

**Review by dmuelle — DISMISSED** [review]
LGTM, thanks

**Comment by njr-11** [comment]
#build
#libby

**Review by jmstephensgit — APPROVED** [review]

**Pull request merged**